### PR TITLE
pass flux-namespace value to the annotation for frontend

### LINKED ISSUE #3329: No flux namespace information in the vcluster objects
The current implementation of the /session_logs endpoint is missing the necessary parameters to be passed by the frontend. This is causing issues with the GitOps Run UI.

We would need to pass only the following parameters from the frontend:

- sessionNamespace
- sessionId
- token

where `clusterName` is `${sessionNamespace}/${sessionId}`

For the flux namespace, the `/session_logs` endpoint would automatically discover it. We would do that by embedding the flux-namespace information into vcluster objects.

### PR DESCRIPTION
**Pull request by chanwit**
Fixes #3329

This PR is to add the `flux-namespace` annotation for vcluster. The `clusterName` has been removed by this PR (as the `clusterName` of a session is actually `${sessionNamespace}/${sessionId}`).

The `/session_logs` endpoint now accepts the following parameter.
- sessionNamespace
- sessionId
- token

Frontend now requires to pass `sessionNamespace` (from the `namespace` annotation), sessionId (from the `sessionName`), and token (empty or value obtained from the next /session_logs call).

Signed-off-by: Chanwit Kaewkasi <chanwit@weave.works>

cc @joshri @opudrovs

** **BREAKING CHANGES for the GitOps Run UI on WGE** **

To test this PR you're required to:
- start a WGE instance by update WGOSS to the branch of this PR
- compile the new `gitops` binary from this branch and create a session with it.
- fix the frontend to pass the above 3 parameters instance of the old ones.

### CODE REVIEW

**Comment by opudrovs** [comment]
I pulled in the same branch in enterprise and see the same issue as described in https://github.com/weaveworks/weave-gitops/issues/3297

A GitOps Run session appears in the list of sessions in the UI, but if clicking on the links to its detail pages after the message `► Waiting for GitOps Run session run-master-fdd0a0b7 to be ready ...` appears, there are still errors on detail pages displayed.

And I am not sure how to passing fluxNamespace is supposed to fix this issue (maybe it fixes a similar issue, but I am not sure about this one).

Screenshots:

Screenshot 2023-01-31 at 10 03 46
Screenshot 2023-01-31 at 10 03 42
Screenshot 2023-01-31 at 10 03 51

File: api/core/core.proto [hunk 1]
```diff
@@ -295,10 +295,9 @@ message ToggleSuspendResourceResponse {
...
 
 message GetSessionLogsRequest {
-    string sessionId   = 1;
-    string token       = 2;
-    string clusterName = 3;
-    string namespace   = 4;
+    string sessionNamespace = 1;
+    string sessionId        = 2;
+    string token            = 3;
 }
```
Comment (chanwit) [inline api/core/core.proto:301]: FYI @joshri @opudrovs

**Comment by makkes** [comment]
Capturing our discussion in standup today: After this got merged, building of the Enterprise UI will break. @joshri will be looking into fixing it today so we don't block the Enterprise release later this week.

**Review by joshri — APPROVED** [review]
I created a new issue to address the links to resources that don't exist yet - it's separate enough from the getLogs request and needs more discussion on how to fix. [#2310](https://github.com/weaveworks/weave-gitops-enterprise/issues/2310)

**Comment by opudrovs** [comment]
@joshri will #2310 be a duplicate of https://github.com/weaveworks/weave-gitops/issues/3297 or will it replace it?

**Comment by joshri** [comment]
Yours is way better so I'm closing mine - MY BAD.

**Comment by opudrovs** [comment]
NO WORRIES!

**Pull request merged**